rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive and evaluate
+applications to the team or alter the composition at any time with approval
+from the NixOS Foundation board. The team's composition and announcements
+should be maintained at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities for the [community
+discussions](https://nixos.org/community/index.html) on behalf of the NixOS
+Foundation. The team should utilize the [NixOS Foundation
+mission](https://nixos.org/community/index.html) and the following statement
+during their duties:
+
+```
+The NixOS Foundation aims to promote participation without regard to gender,
+sexual orientation, disability, ethnicity, age, or similar personal
+characteristics. We want to strive to create and foster community by providing
+an intentionally welcoming and safe environment where all feel valued and cared
+for, and where all are given opportunity to participate meaningfully. The
+Foundation will work with the community in service of this goal.
+```
+
+ref: [twitter](https://twitter.com/grhmc/status/1390775249424338944)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+The initial Moderation Team is defined to be @grahamc, @zimbatm, @domenkozar,
+and @ryantm.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* The moderation team has limited guidance from this RFC on the processes and
+  procedures of the team.
+* This RFC is designed to address a narrow part of a current issue facing the
+  community. Additional RFCs may be needed to address additional concerns.
+* As this is a controversial topic there is a potential this RFC does not have
+  enough detail to be acceptable by the overall community.
+
+# Alternatives
+[alternatives]: #alternatives
+
+* An existing [RFC 98][].
+* Do nothing.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+* Does the team require additional guidance?
+* Does the NixOS Foundation board want to be involved in this manner?
+
+# Future work
+[future]: #future-work
+
+* A potential RFC providing additional guidance and detail for the moderation
+  team's activities and functions.
+* The role of the moderation team could evolve through an effort similar to
+  [RFC 98][] into taking a broader community leadership responsibility as a
+  'Leadership Team' or 'Community Team'.
+* Further evolve our Community Governance by adopting typical governance tools,
+  such as [Contributor Covenant](https://www.contributor-covenant.org/),
+  Statements of Values, and others in order to provide better guiding
+  principles to our community.
```

Comment:
Not sure CC actually says anything useful on the issues that are in-scope for this RFC and left unclear…

Reply:
This statement is an acknowledgement that this RFC is far more limited in scope than RFC98 and that there may be a desire to work on principles/etc in a future RFC. I'm okay either way (no strong opinion), are you recommending we remove the entire bullet point?

Reply:
Well, if the approach here is «minimal useful RFC», I would say that both evolving the team, and calling out specific mixing-everything-together things seems out of the narrowly-carved scope. 

So additional guidance is a good bullet point, and something like «Work on distilling and writing down more specific community norm guidelines» could be another, but evolving the roles and mentioning specific external things is maybe too much.

Although different people ascribe different level of confidence to Future Work section, I guess.

Reply:
The enumeration is mainly aimed at explaining "typical governance tools" through examples. Without any exemplification, it might be a bit harder for the reader to figure out its meaning.